core/src/main/scala/better/files/File.scala

```diff
@@ -113,7 +113,7 @@ class File private (val path: Path)(implicit val fileSystem: FileSystem = path.g
     * @see parentOption
     * @return
     */
-  def parent: File =
+  def parent: File | Null =
```

Comment:
Variables that really can be null and should therefore have a `| Null` type.
`| Null` used: 3
`.nn` used: 3